docs/tutorials/proposals/new-perpetuals-market.md

```diff
@@ -141,6 +141,37 @@ Liquidity monitoring uses the following properties:
 | `timeWindow` | Defines the length of time (in seconds) over which open interest is measured. | 3600 |
 | `scalingFactor` | The target stake scaling factor scales the estimated required liquidity (based on the market's risk model and current market data) to yield the market's target stake. The scaling factor must be a number greater than zero and finite | 10 |
 
+### Mark price configuration
+
+The mark price methodology can be fine-tuned per market. If left blank, the market will default to the [last price method](../../concepts/trading-on-vega/margin.md#last-traded-price).
+
+| Field | Description | Value |
+| ----------- | ----------- | --------- |
+| `decayWeight` | Controls to what extent observation time impacts the weight in the mark price calculation. 0 implies uniform weights. | |
+| `decayPower` | Controls how quickly the weight assigned to older observations should drop. The higher the value, the more weight is assigned to recent observations. |  |
+| `cashAmount` | Used in calculating the mark price from the order book, in asset decimals. | If you expect the market to be highly liquid, use a value the equivalent to 200 USD. If you expect this market to have 10x less liquidity than it would on a high-volume platform, use the USD 20 equivalent. |
```

Comment:
I wonder if we should say USD because it's the settlement asset for the market. OTOH all our markets settle in tUSD for now so... complete accuracy vs easy readability.